tck-dist/userguide/src/main/jbake/content/config.inc

```diff
@@ -41,30 +49,19 @@ and configuration to support your implementation and Arqulillian container imple
 starts up your container and deploys the test wars into it.
 
 
-4.2 Setting up an Environment for the Legacy Jakarta Authentication TCK Against a Compatible Implementation
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-==== 4.2.1 Download the Legacy TCK Contents
+4.3 Setting up an Evironment Against a Jakarta EE {JakartaEEVersion} CI for legacy TCK tests
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-From the tck/old-tck-runner module, execute the following command:
-`mvn generate-resources`
-
-This will unpack the legacy TCK at tck/old-tck-runner/target/authentication-tck.
-
-By default the legacy TCK is retrieved from 
+The legacy tests which are part of the latest Jakarta Authentication TCK use `KEYWORDS` parameter to identify the
+tests that need to be executed for JakartaEE Full and Web profile implementations. The tck/old-tck/run/pom.xml includes
+example of using the profiles `platform` (for JakartaEE Platform - Full profile by default) and `web` (for Web Profile) 
+that sets the appropirate values for `KEYWORDS` in the CI.
 
 ```
-https://download.eclipse.org/jakartaee/authentication/3.0/jakarta-authentication-tck-3.0.0.zip
+For JakartaEE Platform Full profile , KEYWORDS = jaspic_baseline|javaee
+For JakartaEE Platform Web platform , KEYWORDS = jaspic_web_profile
```

Comment:
> Can Keywords be set to javaee or the above mentioned platform?

tck/old-tck/source/pom.xml  has Maven profiles defined for `platform` and `web` which confused me since the word `profiles` is overloaded.